.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2024/10 to 2024/11
+1. AIRAC (2411) - Moved position of EGMD (LDZ) NDB - thanks to @lyraongithub
```

Comment:
```suggestion
1. AIRAC (2411) - Moved position of Lydd (LDZ) NDB - thanks to @lyraongithub
```
Just to be very pedantic and have the name of the NDB, rather than ICAO of nearby airport. (And so you can see how a suggestion works) :)